spec/features/assignment_creation_spec.rb

```diff
@@ -39,17 +47,32 @@ def get_selected_id(finder_var)
     create(:deadline_right, name: 'Late')
     create(:deadline_right, name: 'OK')
   end
+
   describe "creation page", js: true do
     before(:each) do
       (1..3).each do |i|
         create(:course, name: "Course #{i}")
       end
     end
 
+    it "is able to create Review Questionnaire" do
```

Comment:
In the requirement, you do not need to test questionnaire creation. And this is assignment creation test file, not questionnaire test file.